src/main/java/mekanism/common/tile/TileEntityDigitalMiner.java

```diff
@@ -448,8 +448,12 @@ public TileEntity getPullInv() {
     }
 
     public TileEntity getEjectInv() {
```

Comment:
I know it wasn't marked before but we probably should add `@Nullable` to this and the `getEjectTile()`

Reply:
I think that should be left to another PR or changeset. As there are tons of methods missing those type of annotations.